### ISSUE TITLE
Errors when toggling checkboxes in Boss Options

### ISSUE THREAD

**Issue by Road-block** [DeadlyBossMods/DBM-Classic#87]
**Describe the bug**

**Do you have an error log of what happened?**
```
2x FrameXML\OptionsPanelTemplates.lua:151: attempt to call method 'SetValue' (a nil value)
FrameXML\OptionsPanelTemplates.lua:151: in function 'BlizzardOptionsPanel_CheckButton_SetNewValue'
FrameXML\OptionsPanelTemplates.lua:131: in function 'BlizzardOptionsPanel_CheckButton_OnClick'
[string "*:OnClick"]:6: in function <[string "*:OnClick"]:1>
[C]:: ?
```

**To Reproduce**
1. /dbm to show options
2. Select Blackwing Lair > Load Addon
3. Expand to Vael
4. Try to toggle [Show warning when Flame Breath..] or [Yell (with player name) when ..]

**Screenshots**

**Did you try having DeadlyBossMods as the only enabled addon and everything else (especially something like ElvUI) disabled?**

**Which version of DeadlyBossMods are you using?**
1.13.32 (2020/02/10 13:59:44)
**Was it working in a previous version? If yes, which was the last good one?**
N/A
**Additional context**
Options used outside a raid.

**Comment by Road-block** [comment]
The same is true for other Bosses (example [Announce Mortal Strike targets] on Broodlord.

**Comment by MysticalOS** [comment]
Oh I wasn't aware that bad change ever got merged into classic branch, i'll have to fix that and tag another release i guess

**Comment by MysticalOS** [comment]
Fixed in 1.13.33
